### DISCUSSION TITLE
Different output on web platform and webos device

### DISCUSSION THREAD

**Discussion by battistini01** [pavjacko/renative#1115] [Q&A]
Hi, 
I'm getting different launch result between`rnv run -p web` and `rnv run -p webos`. 
I can run without problems with first version, but no with second one.  
I'm not getting errors but the output on my webOS device is not the same of web version running on browser. ( On LG device my app shows only flexn logo and then white page while in browser correctly shows all different contents I wrote). 
This happens with the default template @flexn/create-template-starter template too, without any change after completed the first project initialization.

Anyone can help?

Second question: is possible to launch webOS Simulator and not the old Emulator? rnv is actually searching Emulator, but my mac M1 chip is not supported (https://webostv.developer.lge.com/develop/tools/emulator-installation)

I'm using @flexn/create-template-starter template, "rnv": "0.37.3"

I'd be very thankful if someone could have a call so I can show these problems live

**Comment by thepunisherxx** [comment]
> Hi, I'm getting different launch result between`rnv run -p web` and `rnv run -p webos`. I can run without problems with first version, but no with second one. I'm not getting errors but the output on my webOS device is not the same of web version running on browser. ( On LG device my app shows only flexn logo and then white page while in browser correctly shows all different contents I wrote). This happens with the default template @flexn/create-template-starter template too, without any change after completed the first project initialization.
> 
> Anyone can help?
> 
> Second question: is possible to launch webOS Simulator and not the old Emulator? rnv is actually searching Emulator, but my mac M1 chip is not supported (https://webostv.developer.lge.com/develop/tools/emulator-installation)
> 
> I'm using @flexn/create-template-starter template, "rnv": "0.37.3"
> 
> I'd be very thankful if someone could have a call so I can show these problems live

same here :(

**Comment by pauliusguzas** [comment]
@battistini01 @wwwebgainss0 hello, thanks for using renative.

For the first part this is known bug and is tracked here https://github.com/flexn-io/renative/issues/925
As for webOS simulator support, at the moment it is not supported but version 1.x.x is currently under heavy development which will support it

**Reply by pauliusguzas** [reply]
I see this was answered in ticket therefore closing
